src/status_im/subs.cljs

```diff
@@ -484,7 +484,9 @@
  :<- [:multiaccount]
  (fn [multiaccount]
    (or (get multiaccount :log-level)
-       config/log-level-status-go)))
+       (-> config/log-level
+           name
+           string/upper-case))))
```

Comment:
The log-settings view has a subscription to this function here and previously tried to read from `:multiaccount :log-level`. If there's no log-level in the current multi account, it would fallback to`status-go-log-level`.

^ This is an issue because it would always result in the UI showing a different log level than what is actually configured. That's why I decided to fallback to `config/log-level` instead because that's the same value that's set during app bootstrap.